theme/src/js/components/ProductTabs.js

```diff
@@ -56,19 +60,30 @@ class TabList extends preact.Component {
 		);
 	}
 
-	render({tabList}, {content}) {
+	render({tabList}, {activeId, content}) {
 		return (
 			<div class="row">
 				<div class="col-md-3 products-landing-tablist">
 					<ul class="nav nav-stacked" role="tablist">
 						{tabList.map(
-							tab => (
-								<li class="nav-item" key={tab.id} role="presentation">
-									<button class="btn btn-unstyled nav-link" id={tab.id} onClick={this.handleClick} role="tab" type="button">
-										{tab.name}
-									</button>
-								</li>
-							)
+							tab => {
+								const className = getCN(
+									{
+										'active': tab.id === activeId
+									},
+									'btn',
+									'btn-unstyled',
+									'nav-link'
+								);
+
+								return (
+									<li class="nav-item" key={tab.id} role="presentation">
+										<button class={className} id={tab.id} onClick={this.handleClick} role="tab" type="button">
+											{tab.name}
```

Comment:
I've found that elements with 3 or more attributes are easier to read when we stack the attributes vertically, what are your thoughts on using that pattern?

Reply:
I originally had the attributes in newlines as that is typically the industry pattern and imo easier to read. Especially when I'm adding `aria` attributes, there is suddenly a lot more attributes to read through. See https://github.com/rdai10/zendesk/commit/ecb63779e84a6c084179c912ffc68be75f36bd14. Jon had a different opinion and thought it's easier to read if he treated the html elements in jsx separately as the preact components. 

Should we talk about it together and come to a consensus?

Reply:
I think html elements in jsx code look are fine if we treat them as components and list attributes vertically. I don't think they're interpreted differently and purely for the ease of user viewing. I think we could definitely weigh pros/cons.

Reply:
We can talk about this.  My argument was that if i were to scan the file and quickly glance at everything, differentiating the HTML and the component makes things easier to read.  So I take the same approach we have taken for JSP's, where the HTML markup is written on a single line, where as the taglibs are broken up by attribute on multiple lines.

Reply:
Per offline discussion, we're going to follow the JSP approach where HTML markup is written on a single line, component props are broken up by attribute on multiple lines. @antoniorhee